Makefile

```diff
@@ -18,7 +18,7 @@ pull_test_imgs:
 .PHONY: pull_test_imgs
 
 test: pull_test_imgs
-	go test $(TEST_OPTIONS) -v -failfast -race -coverpkg=./... -covermode=atomic -coverprofile=coverage.out $(SOURCE_FILES) -run $(TEST_PATTERN) -timeout=5m
+	go test $(TEST_OPTIONS) -v -failfast -race -coverpkg=./... -covermode=atomic -coverprofile=coverage.out $(SOURCE_FILES) -run $(TEST_PATTERN) -timeout=25m
```

Comment:
do we really need 25m?

Reply:
No this was just my local dev VM being slow and timing out